src/widgets/note_book.rs

```diff
@@ -96,9 +96,12 @@ impl NoteBook {
         }
     }
 
-    pub fn get_current_page(&self) -> i32 {
-        unsafe {
+    pub fn get_current_page(&self) -> Option<u32> {
+        match unsafe {
             ffi::gtk_notebook_get_current_page(GTK_NOTEBOOK(self.pointer))
```

Comment:
What do you think of using [this conversion implementation](https://github.com/rust-gnome/glib/blob/master/src/translate.rs#L271-281) to express this as
`from_glib(ffi::gtk_notebook_get_current_page(GTK_NOTEBOOK(self.pointer)))`